content/en/blog/2025-09-29-announcing-flux-v2.7.0/index.md

```diff
@@ -0,0 +1,386 @@
+- `OCIRepository.spec.serviceAccountName` for authenticating with AWS ECR, Azure Container Registry and Google Artifact Registry.
+- `ImageRepository.spec.serviceAccountName` for authenticating with AWS ECR, Azure Container Registry and Google Artifact Registry.
+- `Kustomization.spec.decryption.serviceAccountName` for authenticating with AWS KMS, Azure Key Vault and Google KMS.
+- `Kustomization.spec.kubeConfig.configMapRef.name` for authenticating with remote clusters on AWS EKS, Azure AKS and Google GKE.
+- `HelmRelease.spec.kubeConfig.configMapRef.name` for authenticating with remote clusters on AWS EKS, Azure AKS and Google GKE.
+- `Provider.spec.serviceAccountName` for authenticating with Azure DevOps, Azure Event Hub and Google Pub/Sub.
+
+For more details on how to configure object-level workload identity for Flux, see the following docs:
+
+- [AWS workload identity](/flux/integrations/aws/)
+- [Azure workload identity](/flux/integrations/azure/)
+- [GCP workload identity](/flux/integrations/gcp/)
+
+## OpenTelemetry Tracing
+
+Starting with Flux v2.7, users can enable OpenTelemetry tracing for Flux reconciliations
+by configuring a Provider of type `otel`:
+
+```yaml
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Provider
+metadata:
+  name: jaeger
+  namespace: flux-system
+spec:
+  type: otel
+  address: http://jaeger-collector.jaeger:4318/v1/traces
+```
+
+The notification-controller converts Flux events into OTEL spans with proper trace relationships
+based on the Flux object hierarchy. Source objects (`GitRepository`, `HelmChart`, `OCIRepository`, `Bucket`)
+create root spans, while `Kustomization`and `HelmRelease` objects create child spans within the same trace.
+Each span includes event metadata as attributes and uses the alert name and namespace as the service identifier.
+
+{{< gallery match="images/*" sortOrder="desc" rowHeight="150" margins="5"
+thumbnailResizeOptions="900x900 q90 Lanczos"
+previewType="blur" embedPreview=true lastRow="nojustify" >}}
+
+For more details on how to configure OpenTelemetry tracing for Flux, please refer to the
+[notification-controller documentation](/flux/components/notification/providers.md#otel).
+
+## Controller Improvements
+
+- The `GitRepository` API gains supports mTLS for GitHub App authentication.
+- The `Kustomization` API now supports [CEL expressions](/flux/components/kustomize/kustomizations.md#dependency-ready-expression) for extended readiness evaluation of dependencies.
+- The `Kustomization` API gains a new field `.spec.ignoreMissingComponents` for ignoring missing Kustomize components in source.
+- The kustomize-controller now supports global SOPS Age key decryption, allowing centralized management of decryption keys.
+- The kustomize-controller can be configured to cancel ongoing health checks when a new source revision is detected with the `--feature-gates=CancelHealthChecksOnNewRevision=true` flag.
```

Comment:
```suggestion
- The kustomize-controller can be configured to cancel ongoing health checks when a new source revision is detected with the `--feature-gates=CancelHealthCheckOnNewRevision=true` flag.
```

Reply:
(The gate is `CancelHealthCheckOnNewRevision`, not `CancelHealthChecksOnNewRevision`)